src/package/nuspec/Microsoft.NET.Test.Sdk.targets

```diff
@@ -41,6 +41,15 @@
   <PropertyGroup>
     <DebugType Condition="'$(TargetFrameworkIdentifier)' != '.NETCoreApp'">Full</DebugType>
   </PropertyGroup>
+ 
+  <!--
+     Generate config file for test project targeting .NET Framework. This config file has have binding redirect which
+	 is needed at time of running tests.
+   -->
+  <PropertyGroup Condition="'$(TargetFrameworkIdentifier)' == '.NETFramework'">
+    <AutoGenerateBindingRedirects>true</AutoGenerateBindingRedirects>
+    <GenerateBindingRedirectsOutputType>true</GenerateBindingRedirectsOutputType>
```

Comment:
This doesn't seem to be documented in msdn: https://msdn.microsoft.com/en-us/library/2fc472t2(v=vs.110).aspx

Why is it required?